beacon-chain/core/helpers/committee.go

```diff
@@ -594,7 +591,7 @@ func crosslinkCommittees(state *pb.BeaconState, input *shufflingInput) ([]*Cross
 	shuffledIndices, err := Shuffling(
 		bytesutil.ToBytes32(input.seed),
 		state.ValidatorRegistry,
-		input.shufflingEpoch)
```

Comment:
1.) if we pass in shuffling epoch here, it would always be a few epochs behind, I think we should just pass in current epoch here